dev/diffs/3.5.7.diff

```diff
@@ -0,0 +1,3067 @@
+diff --git a/pom.xml b/pom.xml
+index 68e2c422a24..d971894ffe6 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -152,6 +152,8 @@
+     -->
+     <ivy.version>2.5.1</ivy.version>
+     <oro.version>2.0.8</oro.version>
++    <spark.version.short>3.5</spark.version.short>
++    <comet.version>0.11.0-SNAPSHOT</comet.version>
```

Comment:
We have started 0.12.0 development #2584 , please update  comet.version to `0.12.0-SNAPSHOT`.